### PR TITLE
Revision based on peer review feedback

### PR DESCRIPTION
**Pull request by brandmaier**
Response to comments from peer review.

### CODE REVIEW

File: manuscript.Rmd [hunk 1]
```diff
@@ -211,6 +212,7 @@ In sum, we are thankful to @gartner2022responsible for highlighting the importan
 We believe the current proposal should aim to better reflect the distinction between rigor and impact for software, similar to the guidelines proposed to evaluate journal articles.
 To this end, we suggest a more fine-grained assessment of rigor, and put emphasis on the merit statement, in which software authors should argue in how far their project impacted other scientific endeavors.
 Last, we hope to arrive at an evaluation scheme that incentivizes the development of scientific open-source software.
+Note that the requirements of a given academic position should guide the weighting of research software when evaluating and selecting candidates. In some cases, research software development may be central to a position and therefore a requirement (such as when hiring professors of statistics or research methods), but in other cases, we hope that research software development activities at least are considered as research output comparable to published papers.
```
Comment (aaronpeikert) [inline manuscript.Rmd]: Newline after sentence + comma

```suggestion
Note that the requirements of a given academic position should guide the weighting of research software when evaluating and selecting candidates.
In some cases, research software development may be central to a position and, therefore, a requirement (such as when hiring professors of statistics or research methods), but in other cases, we hope that research software development activities at least are considered as research output comparable to published papers.
```

File: manuscript.Rmd [hunk 2]
```diff
@@ -146,12 +145,13 @@ However, we like to point out that the current evaluation schemes yields differe
...
 Above, we already discussed the different ways research software can have an impact --- not only by direct usage, but also by reusing software in other packages.
 Open-source software makes broader impact more likely and is a prerequisite for full transparency and reproducibility [@peikert2021reproducible].
 In addition, many aspects of rigor are impossible to evaluate for closed-source software --- for example, whether it is well-tested or bugs have been fixed.
-Therefore, we propose to penalize software if it does not adhere to an open-source license (those approved in a review process by the Open Source Initiative (https://opensource.org/licenses) by allowing them only half of the total achievable points.
+Giving more weight to open licenses aligns well with the original article's emphasis on open science.
+Therefore, we propose to penalize software if it does not adhere to an open-source license (those approved in a review process by the Open Source Initiative (https://opensource.org/licenses), for example, by allowing them only half of the total achievable points.
```
Comment (aaronpeikert) [inline manuscript.Rmd]: Brackets do not line up.

```suggestion
Therefore, we propose to penalize software if it does not adhere to an open-source license (those approved in a review process by the [Open Source Initiative](https://opensource.org/licenses)), for example, by allowing them only half of the total achievable points.
```

File: manuscript.Rmd [hunk 3]
```diff
@@ -1,6 +1,7 @@
 ---
-title             : "Commentary: 'Responsible Research Assessment II: A specific proposal for hiring and promotion in psychology'"
-shorttitle        : "Responsible research assessment"
+title             : "Assessing rigor and impact in research software"
+#subtitle          : "Commentary: 'Responsible Research Assessment II: A specific proposal for hiring and promotion in psychology'"
+shorttitle        : "Rigor and impact"
```
Comment (aaronpeikert) [inline manuscript.Rmd]:
```suggestion
shorttitle        : "Assessing research software"
```

File: manuscript.Rmd [hunk 4]
```diff
@@ -1,6 +1,7 @@
 ---
-title             : "Commentary: 'Responsible Research Assessment II: A specific proposal for hiring and promotion in psychology'"
-shorttitle        : "Responsible research assessment"
+title             : "Assessing rigor and impact in research software"
```
Comment (aaronpeikert) [inline manuscript.Rmd]:
```suggestion
title             : "Assessing rigor and impact of research software for hiring and promotion in psychology"
```

**Review by aaronpeikert — CHANGES_REQUESTED** [review]

**Review by aaronpeikert — APPROVED** [review]

File: manuscript.Rmd [hunk 5]
```diff
@@ -1,6 +1,7 @@
 ---
-title             : "Commentary: 'Responsible Research Assessment II: A specific proposal for hiring and promotion in psychology'"
-shorttitle        : "Responsible research assessment"
+title             : "Assessing rigor and impact of research software for hiring and promotion in psychology"
```
Comment (Maximilian-Stefan-Ernst) [inline manuscript.Rmd:2]: Finde ich super!

**Review by Maximilian-Stefan-Ernst — APPROVED** [review]

**Review by aaronpeikert — APPROVED** [review]

**Pull request merged**